.github/workflows/build-deploy.yml

```diff
@@ -166,7 +166,9 @@ jobs:
             PUBLIC_IMAGE=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ needs.configure.outputs.IMAGE_TAG }}
             SENTRY_RELEASE="${{ needs.configure.outputs.IMAGE_TAG }}"
             SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }}
-            SENTRY_COMMIT="${{ github.sha }}"
+            SENTRY_ORG={{ TODO }}
+            SENTRY_PROJ={{ TODO }}
```

Comment:
@Timtech4u to specify these values, I will add to the repository secrets. Let me know if I should do otherwise.